emulated_hue/api.py

```diff
@@ -895,24 +892,34 @@ async def __async_get_all_lights(self) -> dict:
             result[light_id] = await self.__async_entity_to_hue(entity, light_config)
         return result
 
-    async def __async_create_local_item(
-        self, data: Any, itemtype: str = "scenes"
-    ) -> str:
-        """Create item in storage of given type (scenes etc.)."""
-        local_items = await self.config.async_get_storage_value(itemtype, default={})
-        # get first available id
-        for i in range(1, 1000):
-            item_id = str(i)
-            if item_id not in local_items:
-                break
-        if (
-            itemtype == "groups"
-            and data["type"] in ["LightGroup", "Room", "Zone"]
-            and "class" not in data
+    async def __normalize_local_item(self, data: Any, itemtype: str) -> None:
```

Comment:
While this may work, can we use this structure instead as it makes the code more readable?
```
def func(value) -> output:
  # Code
  return value
  
result = func(value)
```